src/cli/run.js

```diff
@@ -171,3 +177,15 @@ if (validationFailures) {
     // eslint-disable-next-line flowtype-errors/uncovered
     process.exit(1);
 }
+
+if (config.dumpOperations) {
+    const dumpOperations = config.dumpOperations;
+    const parent = dirname(dumpOperations);
+    if (!existsSync(parent)) {
+        mkdir(parent, {recursive: true});
```

Comment:
I think you probably need `mkdirSync` here.

Assuming `{recursive: true}` is equivalent to `-p` for the `mkdir` program, you might not need to check `existsSync(parent)`.

Reply:
🤦‍♂️